.spelling

```diff
@@ -31,6 +31,7 @@ BpkCarouselItem
 
 BpkChip
```

Comment:
We can probably remove this now that `BpkChip` no longer exists?

Unless spelling complains due to the migration guide

Reply:
Yeah probably best to just leave it